uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/overrides/_contact-actions.scss

```diff
@@ -0,0 +1,6 @@
+// Overrides to remove transparent background
+.page-civicrm-contact-view {
```

Comment:
again we are tying the style to a specific page? Why? Please provide a screenshot

Reply:
Here https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/civicrm/contact/_tooltip.scss#L2 the background is forced to be transparent for element having `#crm-contact-actions-list`. I have added used specific page here as specifying only the `#crm-contact-actions-list {...}` does not override the transparent background in shoreditch.

Of not overriden the background for the contact actions menu will be applied.
<img width="1107" alt="screen shot 2018-01-24 at 2 18 11 pm" src="https://user-images.githubusercontent.com/6307362/35322298-da651e46-0111-11e8-8e23-026a1f45c6b3.png">

Reply:
1. Why was that transparent background applied to `#crm-contact-actions-list` in shoreditch in the first place? Please find out
2. How can it be that it doesn't override? Are we sure we are adding the css of this extension *after* shoreditch's css?

In any case, this cannot pass review, you are hard coupling a style (and a very important one) to a specific page, of a specific CMS. It's not maintainable

Reply:
> Why was that transparent background applied to `#crm-contact-actions-list` in shoreditch in the first place? 

The transparent background is there since the beginning. And honestly, I could not find why it was added in the first place.

> Are we sure we are adding the css of this extension after shoreditch's css?

Yes! `contactactions.css` is loaded after `custom-civicrm.css`.

> How can it be that it doesn't override? 
Since in shoreditch, `#crm-contact-actions-list` (ID) is used to add style it has higher specificity at the same time the transparent background is forced. See: https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/civicrm/contact/_tooltip.scss#L2

But in contact we are trying to override using a class 'crm_contact-actions", it has lower specificity than `#crm-contact-actions-list`. As a result, the background does not get overriden from hrcontactactions extension.

Reply:
I've investigated this further, and it looks to me that this `background: transparent !important` could be replaced by `background-image: none;`, because that's what we are doing here: we're reverting a background image
```css
.ac_results {
    background: transparent url(../i/dropdown-pointer.gif) no-repeat 25px 1px;
}
```

Could you check if doing that solves this specificity issues that we're having